docs/api-docs/webhooks/webhook-events.mdx

```diff
@@ -928,15 +1012,14 @@ Consult the [sites section of the Channel Webhooks Guide](/docs/integrations/web
 |:-------------------------|:------------|
 | store/sku/* | Fires for all store/sku events. |
 | store/sku/created | Fires when a new SKU is created.|
-| store/sku/updated | Fires when a SKU is updated. |
+| store/sku/updated | Fires when you edit a variant globally or for a channel locale as an override. The `context` fields are present for only override updates. |
```

Comment:
@bc-andreadao this is not yet ready, we need to remove this from documentation for now. This work has not yet completed

Reply:
Let's move `store/sku/updated` changes to a separate PR and we will merge it once this functionality is implemented by catalog team